content/feedName.ttl

```diff
@@ -12,4 +12,4 @@
 
       Each feed type will have its own naming convention.
     ''' ;
-    skos:related "feedID".
+    skos:related :feedID.
```

Comment:
Please use `feedId` as per https://github.com/orcfax/glossary/blob/main/CONTRIBUTING.md#style-guide

Reply:
thanks for catching this! have gone back through and made consistent according to our style guide

Reply:
fixed
https://github.com/orcfax/glossary/blob/christian/glossary/content/feedName.ttl